docs/pact-functions.md

```diff
@@ -140,7 +140,7 @@ Top level only: this function will fail if used in module code.
 *app*&nbsp;`(x:<a> -> bool)` *list*&nbsp;`[<a>]` *&rarr;*&nbsp;`[<a>]`
 
 
-Filter LIST by applying APP to each element to get a boolean determining inclusion.
+Filter LIST by applying APP to each element. For each true result, the original value is kept.
```

Comment:
The bane of all new FP programmers. *Does `filter` keep them, or drop them, dammit!?*

Reply:
<sniff>my text is in fact accurate as "_in_clusion" is positive polarity, aligning exactly with boolean values. :P

Reply:
I'm hoping to save a few cycles for our future Pact-writing brothers and sisters.